src/CMakeLists.txt

```diff
@@ -220,7 +220,12 @@ endif(USE_STATIC_LIBS)
 if(COMPILE_SERVER)
   add_executable(marian_server command/marian_server.cpp)
   set_target_properties(marian_server PROPERTIES OUTPUT_NAME marian-server)
-  target_compile_options(marian_server PUBLIC ${ALL_WARNINGS})
+  if(MSVC)
+    target_compile_options(marian_server PUBLIC ${ALL_WARNINGS})
+  else(MSVC)
+      # -Wno-suggest-override disables warnings from Boost 1.69+
```

Comment:
Maybe add a short sentence that this only disables the warning for marian-server, will warn everywhere else (I hope?)

Reply:
Done. Yes, it keeps treating other warnings as errors.